addons/hr_payroll_account/tests/test_hr_payroll_account.py

```diff
@@ -0,0 +1,164 @@
+# -*- coding: utf-8 -*-
+
+import time
+from datetime import datetime, timedelta
+from dateutil import relativedelta
+
+from odoo import fields, tools
+from odoo.tests import common
+
+
+class TestHrPayrollAccountCommon(common.TransactionCase):
+
```

Comment:
no need to create separate class for only setup ... create one single class with two different method.
because here only single test case..